ui/Stars/GrayStar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const GeyStar: React.FC = () => {
```

Comment:
GeyStar ? :thinking: na pewno?
przecież to są 2 takie same komponenty z tylko innym kolorem :)
daj kolor jako props i zrób z tego 1 komponent

Reply:
FullCard otrzymuje teraz propsa `variant` i przyjmuje albo `full` albo `empty` i renderuje się albo na różowo, albo na szaro.